src/point_example.F90

```diff
@@ -3,6 +3,7 @@ program point_example
 
```

Comment:
This file has a lot of changes from the default, including (it looks like) lots of test code for various different rates.  Can we clean this file up substantially?  Perhaps you could move the Bremsstrahlung comparison stuff to its own stand alone program to keep `point_example.F90` clean.  

A clean example of the bremstrahlung kernels would be good though